### PR TITLE
[BEAM-14010] [Website] Add Playground section to the Home page

### PR DESCRIPTION
**Pull request by bullet03**
[BEAM-14010]
- Add a button Try Beam to open the Beam Playground page 
- Add Playground section to index.html and baseof.html

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by bullet03** [comment]
R: @AydarZaynutdinov

File: website/www/site/i18n/home/en.yaml [hunk 1]
```diff
@@ -22,6 +22,8 @@
   translation: "Python Quickstart"
 - id: home-go-quickstart
   translation: "Go Quickstart"
+- id: home-playground
+  translation: "Try Beam"
```
Comment (AydarZaynutdinov) [inline website/www/site/i18n/home/en.yaml:26]:
```suggestion
  translation: "Try Playground"
```

**Review by AydarZaynutdinov — CHANGES_REQUESTED** [review]

**Comment by AydarZaynutdinov** [comment]
R: @pabloem

**Review by AydarZaynutdinov — CHANGES_REQUESTED** [review]

File: website/www/site/layouts/index.html [hunk 2]
```diff
@@ -60,6 +60,24 @@ <h2 class="pillars-title">
   </div>
 {{ end }}
 
+{{ define "playground-section" }}
+<div class="body__contained body__section-nav">
+  <h1>Try Beam Playground</h1>
```
[the remark this answers is not in the document]
Comment (bullet03) [reply website/www/site/layouts/index.html:65]: done

File: website/www/site/layouts/index.html [hunk 3]
```diff
@@ -60,6 +60,24 @@ <h2 class="pillars-title">
...
+  <p>Beam Playground is an interactive environment to try out Beam transforms and examples without having to install Apache Beam in your environment.
+    You can try an Apache Beam examples at <a href="https://frontend-beta-dot-apache-beam-testing.appspot.com/">Beam Playground (Beta)</a>.
```
Comment (pabloem) [inline website/www/site/layouts/index.html:67]:
```suggestion
    You can try an Apache Beam examples at <a href="http://play.beam.apache.org/">Beam Playground (Beta)</a>.
```

Let's use the proper  url right?

File: website/www/site/layouts/index.html [hunk 4]
```diff
@@ -60,6 +60,24 @@ <h2 class="pillars-title">
...
+  </p>
+  <br>
+  <br>
+  <iframe
+          src="https://frontend-beta-dot-apache-beam-testing.appspot.com/embedded?enabled=true&example=SDK_JAVA/PRECOMPILED_OBJECT_TYPE_EXAMPLE/MinimalWordCount&code="
```
Comment (pabloem) [inline website/www/site/layouts/index.html:72]: can we also use the play.beam.apache.org URL?

File: website/www/site/layouts/index.html [hunk 5]
```diff
@@ -232,5 +250,8 @@ <h2 class="calendar-title">
     <div class='ctas_row'>
       <a class="ctas_button" href={{ "/get-started/quickstart-go/" | relLangURL }}>{{ T "home-go-quickstart" }}</a>
     </div>
+    <div class='ctas_row'>
+      <a class="ctas_button" href={{ "https://frontend-beta-dot-apache-beam-testing.appspot.com/" | relLangURL }}>{{ T "home-playground" }}</a>
```
Comment (pabloem) [inline website/www/site/layouts/index.html:254]: same?

File: website/www/site/layouts/index.html (hunk 3 above)
Comment (bullet03) [reply website/www/site/layouts/index.html]: changed

File: website/www/site/layouts/index.html (hunk 4 above)
Comment (bullet03) [reply website/www/site/layouts/index.html]: changed

File: website/www/site/layouts/index.html (hunk 5 above)
Comment (bullet03) [reply website/www/site/layouts/index.html]: changed

**Pull request merged**